libraries/utilities/elasticsearch.cpp

```diff
@@ -63,7 +63,7 @@ bool SendBulk(CURL *curl, std::vector<std::string>& bulk, std::string elasticsea
       return 0;
    }
    else {
-      elog(http_code + "error: Unknown error");
+      elog(std::to_string(http_code) + "error: Unknown error");
```

Comment:
Thank you for working on these warnings. I too had this on my To-Do list. Would you also please add a space just after the quotes? ``" error:...`` instead of ``"error:...``

Line 102 also.